distributionviewer/core/static/css/distributionviewer.styl

```diff
@@ -58,8 +59,6 @@ a {
   }
 
   li {
-    margin-bottom: 10px;
-
     &:last-child {
       margin-bottom: 0;
     }
```

Comment:
Thoughts on this? If it's something we'll need again in the next phase that's fine.